containerize/containerize.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash -e
+
+CWD=${PWD}
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+BASE=
+KEEPCACHE="--exclude-caches-all"
+CLEAN="false"
+DIR="/home/cmsusr"
+FILE="Dockerfile"
+JSON="${SCRIPTPATH}/cache.json"
+TAG="analysis"
+TAR=""
+VCS="--exclude-vcs"
+XRDIR=""
+
+usage(){
+    EXIT=$1
+
+    echo "containerize.sh [options]"
+    echo ""
+	echo "-b [base]           the base image to use (default = ${BASE})"
+	echo "-c                  keep the cached files when making the tarball (default = False)"
+	echo "-C                  cleanup the temporary files when finished making the image (default = ${ClEAN})"
+    echo "-d [dir]            project installation area inside the container (default = ${DIR})"
+    echo "-f [file]           the Dockerfile to use to build the image (default = ${FILE})"
+    echo "-j [json]           path to the json file containing the path to cache (default = ${JSON})"
+    echo "-t [tag]            the tag to use for the image (default = ${TAG})"
+	echo "-T [tar]            path to an existing tarball to use (default = ${TAR})"
+	echo "-v                  include the vcs directories (default = False)"
+	echo "-x [XRDIR]          the path to the tarball on an XRootD accessible device (default = ${XRDIR})"
+    echo "-h                  display this message and exit"
+	echo
+	echo "Examples:"
+	echo "./containerize.sh -t <tag> -b docker://docker.io/cmscloud/cc7-cms"
+	echo "podman run --rm -it -v /cvmfs/cms.cern.ch/:/cvmfs/cms.cern.ch/:ro 4a73e8eb3bf5f2a3ab84e2eedfb314e3489529e2b02a464cd7eeb0d2782091ff"
```

Comment:
should the long hash be replaced with `<tag>`?

Reply:
Yes. This is now fixed.